src/sql/workbench/contrib/notebook/browser/notebookActions.ts

```diff
@@ -308,27 +308,42 @@ export class RunParametersAction extends TooltipFromLabelAction {
 		const editor = this._notebookService.findNotebookEditor(context);
 		// Set defaultParameters to the parameter values in parameter cell
 		let defaultParameters = new Map<string, string>();
+		let emptyParameterCell: boolean;
```

Comment:
any reason why we can't just initialize this to false?

Reply:
I can set it to false.